examples/flows/chat/chat-with-image/chat.jinja2

```diff
@@ -0,0 +1,12 @@
+# system:
```

Comment:
Can't wait to see this PR merged!
Please add README.md and workflows for these new samples

Reply:
I need to check the copy right of the referenced image first